### ISSUE TITLE
Port Request: 1.21

### ISSUE THREAD

**Issue by Yoni664** [Superkat32/YouWereSlain#14]
Hello! I hope I'm not being annoying with this, but are there plans for a port for 1.21? I'm thinking about upgrading to this version but this mod is so essential to me that it would hurt my soul to simply discard it

**Comment by Superkat32** [comment]
Yes! I do plan on porting this mod eventually. I've got quite a few bugs to fix though before that can happen, so it may be a while.

**Comment by Keaton42** [comment]
I'm greatly looking forward to this as well, thank you! This mod is so great for immersion

**Comment by thricebite** [comment]
Hello, is this mod going to be updated for 1.21.1?
